src/docs/patterns/empty-states.mdx

```diff
@@ -1,64 +1,89 @@
 
-<!-- @TODO switch empty-state links over to the "Component Recommendations style" instead of a "Interactive Code example" once we have one designed. -->
+### Element-level
 
-<DemoCard
-    repository={'empty-states'}
-    angular={'https://pxblue-components.github.io/angular/?path=/story/components-empty-state--with-full-config'}
-    react={'https://pxblue-components.github.io/react/?path=/story/components-empty-state--with-full-config'}
-    ionic
-    reactNative
-/>
+While less common, empty states can also be used as an indicator for individual sections of a page rather than the entire page (such as a single card having no data on a dashboard / overview screen). In these cases, it may be appropriate to use a smaller icon and text depending on the available space.
 
-Empty states can be used in a variety of ways in your application. They can alert users to empty data or search results, lack of permissions, future features, or errors.
+<Divider />
 
-You can refer to the Material Design Empty States guidelines for general usage instructions.
+# Design Specifications
 
-Empty states should include a large icon or graphic, centered on the screen, followed a brief message. In some cases, you may wish to use a large graphic as the background of the entire page as well (such as for features that will be coming soon).
+<ImageGrid images={[Specs1Image, Specs2Image]} regularWidth />
+<Divider />
 
-If necessary, additional explanatory text may follow. The text on the screen should clearly indicate why there is no data to display, and what (if anything) can be done to make data available. The text should be minimal, but enough to convey the necessary message.
+# Developers
 
-If the data is empty but could be populated by a user action (such as adding a device to an empty device list), there should be a call-to-action button that will allow the user to add data.
+Use the following components to implement this pattern:
+
+**Angular** and **Ionic** - @pxblue/angular-components:
+
+-   [Empty State](https://pxblue-components.github.io/angular/?path=/info/components-empty-state--readme)
+
+**React** - @pxblue/react-components:
+
+-   [Empty State](https://pxblue-components.github.io/react/?path=/info/components-empty-state--get-read-me-story)
+
+**React Native** - @pxblue/react-native-components:
 
-> **NOTE:** While less common, empty states can be used in sections of a page (rather than the entire page), such as a single card having no data. In these cases, the same guidelines should be followed, but may require slightly different formatting depending on the layout of the page.
+-   [Empty State](https://pxblue-components.github.io/react-native/?path=/info/components-documentation--empty-state)
```

Comment:
We should update the developers section in /patterns/appbar to use this format as well?

Reply:
That's the next story on my list